content/en/users/aai/check-in/faq/_index.md

```diff
@@ -9,6 +9,20 @@ description: >
   Most frequent questions about EGI Check-in
 ---
 
+## Getting help
+
+Requests for technical assistance and other issues related to Check-in can be
+obtained by sending an email to:
+
+**checkin-support@mailman.egi.eu**
```

Comment:
```suggestion
> **checkin-support@mailman.egi.eu**
```

Reply:
Should highlight a bit the email and address the linting issue: https://github.com/EGI-Federation/documentation/actions/runs/4173190168/jobs/7225222363#step:4:4738